README.md

```diff
@@ -30,12 +30,10 @@ To learn how to edit and build this repo's content, please refer to
 
 ## Versions and releases
 
-Istio maintains three variations of its public site.
+Istio maintains two variations of its public site.
 
 - [istio.io](https://istio.io) is the main site, showing documentation for the current release of the product.
-
-- [archive.istio.io](https://archive.istio.io) contains snapshots of the documentation for previous releases of the product.
-This is useful for customers still using these older releases.
+[istio.io/archive](https://istio.io/archive) contains snapshots of the documentation for previous releases of the product. This is useful for customers still using these older releases.
```

Comment:
A few lines down (not exactly sure how to select it), it has `All three sites`. This should say `Both sites`?